src/AutoVizuA11y.tsx

```diff
@@ -134,6 +147,15 @@ const AutoVizuA11y = ({
 
 	let componentId = useAutoId();
 
+	useEffect(() => {
```

Comment:
question: shouldn't this (focusing on the modal after opening the guide) be the responsibility of the ShortcutGuide itself? And therefore be in its own component?
let me know if you need me to clarify this